src/test/ui/const-generics/defaults/default-annotation.rs

```diff
@@ -13,7 +13,6 @@ pub struct ConstDefaultUnstable<const N: usize = 3>;
 
 #[stable(feature = "const_default_unstable", since="none")]
 pub struct ConstDefaultStable<const N: usize = {
-    #[stable(feature = "const_default_unstable_val", since="none")]
```

Comment:
Why was this removed?

Reply:
Stability attributes on expressions do nothing, and I wanted to keep this test warning-free.